contentful/apps
# Mjg/feat/add hosted actions

## Purpose



This adds MVP functionality for triggering amplify builds from a user supplied webhook using hosted app actions.

view the action [here](https://github.com/contentful/apps/blob/mjg/feat/add-hosted-actions/apps/aws-amplify/actions/amplify.ts)

## How to test

With access:

1.) visit the amplify staging space [here](https://app.contentful.com/spaces/b7sxrca414e5/environments/master)

2.) make sure the Amplify app is installed (staging)

3.) Navigate to a piece of content, and click `Build now` button to initiate a build

4.) Observe build status in the Amplify console
